.gitignore

```diff
@@ -1,6 +1,6 @@
 HELP.md
 target/
-src/main/resources/application-**.properties
+src/main/resources/application.properties
```

Comment:
If you put this file on gitignore, you will have to warn your team everytime someone is adding a new env variable.